src/store/Account.ts

```diff
@@ -86,6 +89,10 @@ function initialState(): AccountState {
     updateAccountAssetsSubscription: null,
     fiatPriceAndApyObject: {},
     fiatPriceAndApyTimer: null,
+    referralRewards: {
```

Comment:
If you prefer to use default value in initial state, i guess you should update `AccountState `with `referralRewards: <ReferrerRewards>` because i can not see the cases, when `referralRewards` will be `null`